components/hero/hero.js

```diff
@@ -6,8 +6,14 @@ const Hero = () => (
+          </svg>
+        </a>
```

Comment:
Thank you! One note: [Axe](https://www.deque.com/axe/browser-extensions/) is flagging the link for [not having discernible text](https://dequeuniversity.com/rules/axe/4.3/link-name?application=AxeChrome). We can just move the visually hidden span of text on [line 8](https://github.com/sparkbox/apprentices-page-next/pull/27/files#diff-d71fd56f1b275a7aae289f23e8304113aa7b1b213c4f4c4eefca36f9d7f28e33R8) inside of the `a` tag.